packages/opentelemetry/lib/tracing/exporter.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable no-unused-vars */
+
+import { JaegerExporter } from '@opentelemetry/exporter-jaeger';
+import { ZipkinExporter } from '@opentelemetry/exporter-zipkin';
+import { PrometheusExporter } from '@opentelemetry/exporter-prometheus';
+import { ConsoleSpanExporter, SpanExporter } from '@opentelemetry/tracing';
+
+
+const DEFAULT_SERVICE_NAME = 'appium';
+
+const AVAILABLE_EXPORTERS = {
+  JAGGER: 'jagger',
+  ZIPKIN: 'zipkin',
+  PROMETHEUS: 'prometheus',
+  CONSOLE: 'console'
+};
+
+/**
+   * factory to create exporter instance for a given exporter type and optional config
+   * @param {string} exporterType
+   * @param {Object} config
```

Comment:
does this config have some predefined structure?

Reply:
Every exporter has different structure depending on the exporter. For eg 
https://github.com/open-telemetry/opentelemetry-js/blob/7fa4ff70d1aad5875cb14c6bff99c838228447bb/packages/opentelemetry-exporter-jaeger/src/types.ts#L20
, can add all 3 typedef